tests/system-broker/tests/main_test.go

```diff
@@ -1,26 +1,33 @@
 package tests
 
 import (
+	"github.com/kyma-incubator/compass/tests/pkg/clients"
+	"github.com/kyma-incubator/compass/tests/pkg/config"
+	"github.com/kyma-incubator/compass/tests/pkg/testctx"
 	"os"
 	"testing"
 
-	"github.com/kyma-incubator/compass/tests/system-broker/pkg"
-
 	"github.com/sirupsen/logrus"
 )
 
-var config pkg.Config
+var testCtx *clients.SystemBrokerTestContext
 
 func TestMain(m *testing.M) {
 	logrus.Info("Starting System Broker Tests")
 
-	var err error
-	config, err = pkg.ReadConfig()
+	cfg := config.SystemBrokerTestConfig{}
+	err := config.ReadConfig(&cfg)
 	if err != nil {
 		logrus.Errorf("Failed to read config: %s", err.Error())
 		os.Exit(1)
 	}
 
+	testCtx, err = clients.NewSystemBrokerTestContext(cfg)
+	if err != nil {
+		logrus.Errorf("Failed to create test context: %s", err.Error())
+		os.Exit(1)
+	}
+	testctx.Init()
```

Comment:
Just to double check this is needed in order to use fixtures? I would suggest to change it to e "real" `init()` function that is called when package is imported for instance, this way you can remove all the `testctx.Init()` occurrences.